code/modules/mob/say.dm

```diff
@@ -2,6 +2,8 @@
 #define ILLEGAL_CHARACTERS_LIST list("<" = "", ">" = "", \
 	"\[" = "", "]" = "", "{" = "", "}" = "")
 
+#define USABLE_DEAD_EMOTES list("*flip", "*spin")
```

Comment:
Since this is only used in this file, you should `#undef` it somewhere after use. Unless you think this may be used in other files at some point.